app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    Person.people.clear()
```

Comment:
why? 😅

Reply:
for data update: If create_person_list is called multiple times with different data, clearing Person.people allows to remove previous data to avoid mixing old and new data.